client/src/document/baseline-indicator.scss

```diff
@@ -0,0 +1,151 @@
+@use "../ui/vars" as *;
+
+$browsers: "chrome", "edge", "firefox", "safari";
+
+.baseline-indicator {
+  --baseline-bg: var(--baseline-unsupported-bg);
+  --baseline-engine: var(--baseline-unsupported-engine);
+  --baseline-mark: var(--baseline-unsupported-mark);
+
+  background: var(--baseline-bg);
+  border-radius: 0.25rem;
+  margin: 1rem 0;
+
+  &[open] {
+    summary .icon-chevron {
+      transform: rotate(180deg);
+    }
+  }
+
+  &.supported {
+    --baseline-bg: var(--baseline-supported-bg);
+    --baseline-engine: var(--baseline-supported-engine);
+    --baseline-mark: var(--baseline-supported-mark);
+  }
```

Comment:
🟢 I'd suggest keeping this block even closer to the default values above.

Reply:
I did, but stylelint wasn't happy, gonna have to change back to how it was before.